config/logging-test.properties

```diff
@@ -0,0 +1,5 @@
+handlers=java.util.logging.ConsoleHandler
+
+# Tests level of output - change to INFO/WARNING to have more logs for example.
+# This set the level to SEVERE to all the classes using java.util.logging logger when running tests
+java.util.logging.ConsoleHandler.level=SEVERE
```

Comment:
unfortunately third party libraries use logging instead so you need to add the level property to cover this logger.